packages/server/src/main/resources/application.properties

```diff
@@ -133,7 +133,7 @@ server.error.include-stacktrace=never
 ########################################################################################################################
 # AWS credentials configuration
 ########################################################################################################################
-aws-url=http://s3.t1.uncharted.software
+aws-url=
```

Comment:
Can we remove it instead of empty, if it is defined in all other profiles?

Reply:
it needs to be empty for aws to work